### ISSUE TITLE
Create Squirrel Tool to Block Users from Proofreading Activities

### ISSUE THREAD

**Issue by wf49670** [DistributedProofreaders/dproofreaders#1171]
Occasionally we need to block a user from auto-granted proofreading activities such as P1 or P2. Today this is done by running several manual commands via MySQL to directly update the DB. The commands are documented in SAWiki. https://www.pgdp.net/sawiki/index.php/Block_a_user_from_specific_site_activities

This is a potentially risky manual operation that could be done more safely using a script.

**Comment by srjfoo** [comment]
Agreed. Initially, I think we left it as a manual process to force us to think carefully before doing a block. Definitely safer to do it via a script.

**Comment by srjfoo** [comment]
Agreed. Initially, I think we left it as a manual process to force us to think carefully before doing a block. Definitely safer to do it via a script. However, consider whether it should go in as a `noncvs` script instead of in `dproofreaders`.
